src/FilterMonitor.js

```diff
@@ -10,10 +10,11 @@ export default class FilterMonitor extends Component {
     blacklist: PropTypes.array
   };
 
-  isFiltered(action) {
+  isFiltered(actionType) {
+    var type = actionType || '';
```

Comment:
I see, yeh you're right. Neither should Redux DevTools be safe guarding against it though. I think the problem lies with redux sagas as I believe that is what is tripping up. It is something to do with the effects actions maybe. Please could this small fix go in in the meantime so our team doesn't have to all implement this in their node modules?